students/nicole/lab/drills/functions.js

```diff
@@ -69,17 +69,62 @@
 
 // Create a function that returns the sum of three numbers
 
+function numSum(numOne, numTwo, numThree){
+  return numOne + numTwo + numThree  
+};
+
+numSum(4, 4, 4);
+
 // Create a function that multiplies two numbers 
 
+function numTip(parameterOne, parameterTwo){
+  return parameterOne * parameterTwo
+};
+
+numTip(2, 2);
+
 // Create a function that divides two numbers
 
+function numDiv(parameterThree, parameterFour){
+  return parameterThree / parameterFour
+};
+
+numDiv(24,2);
+
 // Create a function that returns the data type of a variable
 // ## HINT: use the typeof keyword
+let myAge = 34;
+
+function retVar(paramOne){
+   return typeof(paramOne);
```

Comment:
Did you test this code to see if it works?

`typeof` isn't a function, it's a keyword, so you don't invoke it with `()` You use it like this: `typeof variableName`

And